### DISCUSSION TITLE
Issue Building Discord Bot with Docker on Raspberry Pi 5

### DISCUSSION THREAD

**Discussion by ByMarvster04** [eritislami/evobot#1591] [General]
Hello Developer Team,

I hope you're having a great day! I wanted to inform you about an issue I encountered while trying to build and run the Discord bot on my Raspberry Pi 5 using Docker.

Here's the error I'm getting in the logs:

root@raspberrypi:/home/pi# docker run -e "TOKEN=my discord token 🙂 " eritislami/evobot

evobot@2.8.1 prod
npm run build && node ./dist/index.js

evobot@2.8.1 build
tsc --build

node:internal/process/promises:279
            triggerUncaughtException(err, true /* fromPromise */);
            ^

ConnectTimeoutError: Connect Timeout Error
    at onConnectTimeout (/home/evobot/node_modules/undici/lib/core/connect.js:182:24)
    at /home/evobot/node_modules/undici/lib/core/connect.js:129:46
    at Immediate._onImmediate (/home/evobot/node_modules/undici/lib/core/connect.js:170:9)
    at processImmediate (node:internal/timers:466:21) {
  code: 'UND_ERR_CONNECT_TIMEOUT'
}

Other projects typically run smoothly in Docker on this setup, but I can't quite understand or resolve this error. Could you please assist me in identifying the problem and suggesting a solution? I would greatly appreciate your help!

Thank you very much in advance for your support.

Best regards,
ByMarvster04

**Comment by eritislami** [comment]
The `prod` command builds and runs the code, what you are seeing is after that and it's a network timeout problem, not directly related to the codebase. Hard to say why that is happening.
